src/main/java/com/xeeva/navigation/MainNav.java

```diff
@@ -255,8 +255,7 @@ public void perform_CartItemVerifications(String UnitPrice,String UOM,String Qua
 	 **/
 	public boolean verifyCartValue(String ItemType){
 		boolean statsuFlag = false;
-		// lblCartValue element is not finding with page,element,script timeouts
-		Sleeper.sleep(10000);
+		Sleeper.sleep(15000);
 		TestReporter.logStep("Cart Having [" + lblCartValue.getText()+"] Items!!");

```

Comment:
15 seconds is long time to hard code a wait.  Can we revisit this and find a way to do a dynamic wait?  Let me know if you want me to take a look and help